jupyter-js-widgets/package.json

```diff
@@ -54,8 +54,7 @@
     "webpack": "^1.12.11"
   },
   "dependencies": {
-    "backbone": "1.2.0",
-    "bootstrap": "^3.3.5",
+    "backbone": "^1.2.3",
```

Comment:
As per discussion in person, we must pin backbone to `1.2.0`. We have noticed a breaking change between 1.2.0 and 1.2.2. (This specific regression seemed to have been fixed in 1.2.3 though) but the crux is that backbone folks admittedly don't follow semver and we can't use these types of loose version requirement with it.